_posts/2019-04-17-dao-launch-third-anniversary.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: "Bisq DAO Launches with Bisq v1.0"
+author: Steve Jain
+excerpt: "After years in the making, the Bisq DAO has launched on mainnet with the v1.0 release of the Bisq software, just days before Bisq celebrates 3 years in production. <br><br>"
+---
+
+<hr>
+
+_Steve contributes documentation, copywriting, speaking, media correspondence, and more to the Bisq network. Follow him [here](https://jain.io)._
+
+_If you'd like to contribute a post, please reach out on [Slack](https://bisq.network/slack-invite) or [on our forum](https://bisq.community/t/call-for-blog-writers/7040)!_
+
+<hr>
+
+On Monday, Bisq hit the most significant landmark in its 5-year history: it launched v1.0 of its software.
+
+Version 1 is always a big milestone for any software project, but in Bisq's case it was extra-special: it included the mainnet launch of the Bisq DAO, which fundamentally changes the governance of the Bisq network itself.
+
+Bisq stakeholders (traders and contributors) are now in charge. The Bisq founder role [has been dissolved](https://github.com/bisq-network/roles/issues/1#issuecomment-478204839).
+
+The Bisq DAO has been in the works since work on Bisq begun back in 2014, but it's been iterated conceptually and technically many times since then to ensure it can achieve truly decentralized management and funding of the Bisq network.
+
+[The constraints](https://twitter.com/bisq_network/status/1103293547711746050) were not trivial: there could be no centrally-owned wallets, no legal entities, no investors, etc.
+
+Governance is always a tough thing to tackle, especially for a non-protocol project where people are needed to consistently deliver development, support, and other services, but the Bisq DAO seems to offer a promising solution. It offers a system of incentives that pushes all stakeholders to make the project sustainable simply by acting in their own best interest. The resulting dynamic challenges many social, economic, and organizational dynamics in significant ways, which we've started to cover in [this series](https://bisq.network/blog/dao-benefits-funding/).
+
+Early indications are encouraging: about 150 trades used BSQ to pay trading fees in the first 24 hours it was live.
```

Comment:
150? Where do u have that number? I see 64 trades so far in the trades stats

Reply:
dao > facts & figures > bsq transactions > number of fee payments transactions shows 207 right now, and it was about 150 late yesterday morning.

i could be wrong...am i understanding that number correctly?

Reply:
actually 129 is the exact number, just counted from the block explorer from genesis up to 11am april 16.

Reply:
There are 65 BSQ/BTC trades atm.
But I think that m52go speaks about trades using BSQ to pay the fees. And that is indeed nice.